Tests/Reqnroll.SystemTests/Portability/PortabilityTestBase.cs

```diff
@@ -42,6 +42,8 @@ private void RunSkippableTest(Action test)
         {
             if (!new ConfigurationDriver().PipelineMode)
                 Assert.Inconclusive(ex.ToString());
+            else
+                throw;
```

Comment:
This change seems significant; what does allowing the exception to bubble here achieve?

Reply:
While investigating the changes for the pull request (PR), I discovered that the .NET 7 runtime and SDK are no longer installed on the runners.
However, our .NET 7 tests are successful (green). This is because the DotNetSdkNotInstalledException is caught here and not rethrown. With the rethrow, the tests fail when the .NET SDK is not installed, as desired.